Lib/fontMath/test/test_mathInfo.py

```diff
@@ -213,6 +213,27 @@ def test_round(self):
                 expectedValue = _roundNumber(value)
             expected[attr] = expectedValue
         self.assertEqual(sorted(expected), sorted(written))
+
+    def test_sub_undefined_number_list_does_nothing(self):
+        self.assertIn("postscriptBlueValues", _numberListAttrs)
+
+        info1 = _TestInfoObject()
+        info1.postscriptBlueValues = None
+        m1 = MathInfo(info1)
+
+        info2 = _TestInfoObject()
+        info2.postscriptBlueValues = [1, 2, 3]
+        m2 = MathInfo(info2)
+
+        m3 = m2 - m1
+
+        self.assertEqual(m3.postscriptBlueValues, [1, 2, 3])
```

Comment:
Ugh, I had expected a result of `None` here. This used to have a result of 0, which is buggy, but at least had an intention. I see where the code does this (the conditional `setattr()`), but I'm not confident now we have the right behavior.

Reply:
argh. I've pushed a [0.5.1](https://github.com/robotools/fontMath/releases/tag/0.5.1) release already!

but note that currently if you subtract two number-list attributes of different length, you also get `None`, which is equivalent of not setting the attribute (i.e. keep the one from the first operand). See `_processMathOneNumberList` method.
When one of the list attribute is undefined, its length is also by definition _different_ from the other well defined number-list operand, so the result is similarly `None` or do nothing.

Reply:
If a list was not interpolatable you'd get wrong results anyway, which may be defendable because "those lists need to match".

But now you _also_ get wrong results if one operand is missing, and given how the result will be used (as a delta), and how apparently the previous `0` accidentally did the right thing in some environments, I'm fairly confident we'll now see a regression in real use cases where only the default master specifies (things like) `postscriptBlueValues`.

Reply:
...and other masters leave it blank.